go/nbs/table_set.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package nbs
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/attic-labs/noms/go/d"
+)
+
+func newFSTableSet(dir string) tableSet {
+	return tableSet{p: fsTablePersister{dir}}
+}
+
+// tableSet is an immutable set of persistable chunkSources. Prepending a
+// memTable to an existing tableSet compacts the memTable and returns a new
```

Comment:
maybe move the second sentence to `Prepend` and add a corresponding comment to `Union`?